### DISCUSSION TITLE
Possible to get multiple counts returned?

### DISCUSSION THREAD

**Discussion by justinhandley** [prisma/prisma#7524] [Q&A]
I have a scenario where I'd like to be able to return multiple counts on an object.

For instance, I have an audio file.  Many people like it.  Many people have listened to it.  I'd like to show both without multiple queries. 

So instead of something like...

_count { plays: true }

either have that somehow return multiples

_count { plays: true, favoritedBy: true }

Or nested

{ plays: {_count: true}, favoritedBy: {_count: true}}

Is this difficult?  Does anyone have workarounds they use if they want two or three counts in a query?

**Comment by ryands17** [comment]
@justinhandley 👋 

This is possible currently as long as they are in relations. For e.g. given this schema:

```prisma
generator client {
  provider        = "prisma-client-js"
  previewFeatures = ["orderByRelation", "selectRelationCount"]
}

model User {
  id       Int       @id @default(autoincrement())
  name     String
  posts    Post[]
  profiles Profile[]
}

model Post {
  id     Int    @id @default(autoincrement())
  title  String
  User   User?  @relation(fields: [userId], references: [id])
  userId Int?
}

model Profile {
  id     Int   @id @default(autoincrement())
  User   User? @relation(fields: [userId], references: [id])
  userId Int?
}
```

I can get the `posts` and `profile` count as follows:

```ts
await prisma.user.findMany({
    include: { _count: { select: { posts: true, profiles: true } } },
})
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
